### DISCUSSION TITLE
WiFi Connection to Internet that requires user input (beyond password authentication)

### DISCUSSION THREAD

**Discussion by JonathanNash21** [iRobotEducation/create3_docs#238] [Questions] [answered]
Hello,

I'm trying to connect my Create3 to my internet, but in order to fully connect, the wifi connection re-routes traffic to a confirmation page where the user must click an acceptance to usage policy button. This is fine on something like a laptop, but I can't figure out how to get the Create3 connected, as I can't accept since I don't have access to a user interface. Is there some way to have the Create3 auto-accept terms like these, or to be able to attach the Create3 to a monitor so I can manually accept them to get it fully connected to the WiFi? Or, at the very least, is there a simple way to find the MAC address for the Create3 so I can contact the network admins and have them add the device to the wifi?

**Comment by shamlian** [comment, marked as answer]
The MAC address for the Create 3 is available on the "About" page in the web server. If you connect over soft AP, you'll be able to find it in the "ifconfig" section. [Here's a link to the docs.](https://iroboteducation.github.io/create3_docs/webserver/about/#ifconfig)

**Reply by JonathanNash21** [reply]
Much appreciated, thank you for the quick response!
